akka-sample-javaslang/src/main/java/sample/javaslang/Master.java

```diff
@@ -0,0 +1,54 @@
+package sample.javaslang;
+
+import akka.actor.Props;
+import akka.actor.UntypedAbstractActor;
+import javaslang.collection.List;
+import javaslang.collection.Stream;
+
+import static sample.javaslang.Minion_StoppedPatterns.*;
+
+import static javaslang.API.*;
+import static javaslang.Predicates.*;
+
+public class Master extends UntypedAbstractActor {
+
+  private final Maze maze;
+
+  private List<Minion.Stopped> results = List.empty();
+
+  public Master(Maze maze) {
+    this.maze = maze;
+  }
+
+  public void onReceive(Object message) {
+    Match(message).of(
+      Case(instanceOf(Start.class), s -> {
+        Stream.range(0, s.count).forEach(c ->
+          getContext().actorOf(Props.create(() -> new Minion(this.maze)))
```

Comment:
We should still use the convention with static `props` factory method. There is actually a bug here. `this.maze` is accessed from another thread, when `Minion` is created.